### PR TITLE
Few improvements for easier hacking

### PR DESCRIPTION
**Pull request by LVerneyPEReN**
Hi,

Here are a few improvements for contributions/hacking.

* First, I added some extra doc and made sure it is easy to run the console scripts without installing the package, for easier contribution.
* Second, it is sometimes useful to capture traffic remotely, on an OpenWRT-based router for instance. I added an argument so that one can overload the capture command to support this setup.

Best,

### CODE REVIEW

File: pirogue_cli/network/packet_capture.py [hunk 1]
```diff
@@ -42,12 +50,13 @@ def __check_user_rights():
...
         log.info(f'⚡ Starting network interception...')
-        TcpDump.__check_user_rights()
+        if not self.has_user_provided_cmd:
+            TcpDump.__check_user_rights()
         try:
             self.process = subprocess.Popen(
                 self.capture_cmd,
                 shell=True,
-                stdout=subprocess.PIPE,
+                stdout=open(f'{self.output_dir}/{self.pcap_file_name}', 'w'),
```
[the remark this answers is not in the document]
Comment (LVerneyPEReN) [reply pirogue_cli/network/packet_capture.py:59]: Should have been fixed, tested on a regular PiRogue setup.

**Comment by U039b** [comment]
Thank you very much! Should I merge it now, or your PR is still a WIP?

**Comment by LVerneyPEReN** [comment]
If this is fine for you (my main concerns are explicit dependencies in the `setup.py` etc., which I hope won't affect the way you build and distribute the `apt` package), I think this one is ready to get merged.

**Comment by U039b** [comment]
I have integrated your changes along with others. Listing explicit dependencies in `setup.py` broke things. So, to allow both installation methods (`apt` and `pip`) we had to introduce [a patch](https://github.com/PiRogueToolSuite/pirogue-cli/blob/main/debian/patches/0001-disable-requirements.diff) which is applied during the build of the Debian package.

**Comment by LVerneyPEReN** [comment]
Thanks!

**Pull request closed without merging**